transformer/dockerfilegenerator/dotnetcoredockerfilegenerator.go

```diff
@@ -438,9 +438,9 @@ func (t *DotNetCoreDockerfileGenerator) TransformArtifact(newArtifact transforme
 			buildOutputDir = filepath.Clean(common.GetUnixPath(configuration.PropertyGroups[idx].OutputPath))
 		}
 		relBuildOutputDir := filepath.Join(buildOutputDir, targetFramework, defaultBuildOutputSubDir)
```

Comment:
Dont' we have to do it here too?

Reply:
No, because in that case we are copying from the local filesystem which should use the OS specific path separator

Reply:
have changed to unix paths